src/main/java/org/example/bookstore/controller/BookController.java

```diff
@@ -20,33 +22,38 @@
 import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestController;
 
+@EnableMethodSecurity
 @Tag(name = "Books management", description = "Endpoints to manage books")
 @RestController
 @RequestMapping("/books")
 @RequiredArgsConstructor
 public class BookController {
     private final BookService bookService;
 
+    @PreAuthorize("hasRole('USER')")
     @Operation(summary = "Get all books", description = "Get a list of available books")
     @GetMapping
     public List<BookDto> getAll(Pageable pageable) {
         return bookService.findAll(pageable);
     }
 
+    @PreAuthorize("hasRole('USER')")
```

Comment:
check methods with USER hasRole, when you authenticate with ADMIN. I think you won't be able to see books from admin

Reply:
I do, since the user has a set of roles, I simply add the user role in liquibase script if I understood your comment correctly)